app/views/govuk_publishing_components/components/docs/document_list.yml

```diff
@@ -211,6 +211,23 @@ examples:
         metadata:
           public_updated_at: 2017-07-19 15:01:48
           document_type: 'Statistical data set'
+  without_top_border_on_first_list_element:
+    description: Several interfaces across GOV.UK benefit from the semantics of the document list but have their own bespoke designs, sometimes meaning that the visual border element doesn't gel with said interface. Removing it using the below attribute allows for cleaner visual fidelity in this instances.
```

Comment:
Very small nitpick, would just change "in this instances" to "in these instances"